### DISCUSSION TITLE
PropertyChangedMode == PropertyChangedModes.Windows for Blazor?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [MarimerLLC/csla#2831] [Questions]
This is how the Blazor example is configured. The reason I ask is it needs to be `Windows` for the following to work:

```cs
 public static readonly PropertyInfo<bool> IsWidgetProperty = RegisterProperty<bool>(nameof(IsWidget));
 public bool IsWidget
 {
     get { return ReadProperty(IsWidgetProperty); }
     set { SetProperty(IsWidgetProperty, value); }
 }

 protected override void OnPropertyChanged(IPropertyInfo propertyInfo)
 {
     base.OnPropertyChanged(propertyInfo);

     if (propertyInfo == IsWidgetProperty)
     {
         // IsWidget has changed
     }
 }
```

**Comment by rockfordlhotka** [comment]
There's another thread from the past few days discussing the way PropertyChanged affects Blazor, including an experiment using `Windows` for `PropertyChangedMode`. It sounded like that experiment was showing promise, and you might want to post in that thread to see how it is going.

It may be that `Windows` is the correct setting for Blazor - not just for what you are seeing, but for performance overall.

**Reply by michaelcsikos** [reply]
Thanks, I did do a search before posting the question, but I couldn't find any recent threads. (Still can't, in fact.)

**Reply by rockfordlhotka** [reply]
https://github.com/MarimerLLC/csla/discussions/2823

**Reply by rockfordlhotka** [reply]
Also now an issue: https://github.com/MarimerLLC/csla/issues/2833
